docs/v2-MigrationGuide.md

```diff
@@ -4,11 +4,11 @@
 
 ### Android
 
-* Merchant's app theme must be decendent of `Theme.MaterialComponents` to operate with "instant" payment components (ex. Paypal, Klarna). Example:
+* Merchant's app theme must be descendent of `Theme.MaterialComponents` to operate with "instant" payment components (ex. Paypal, Klarna). Example:
 ```xml
     <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar"> 
 ```
-* `adyenReactNativeRedirectScheme` was deprecared. Use any [intentFilter](https://developer.android.com/guide/components/intents-filters). SDK will still provides `returnUrl` value inside of the `onSubmit.data` in case it is needed, be cautious to not override it.
+* `adyenReactNativeRedirectScheme` was deprecated. Use any [intentFilter](https://developer.android.com/guide/components/intents-filters). SDK will still provides `returnUrl` value inside of the `onSubmit.data` in case it is needed, be cautious to not override it. Also, for Android Drop-in `await AdyenDropIn.getReturnURL()` can be used to extract a `returnUrl`.
```

Comment:
```suggestion
* `adyenReactNativeRedirectScheme` was deprecated. Use any [intentFilter](https://developer.android.com/guide/components/intents-filters). SDK will still provide `returnUrl` value inside of the `onSubmit.data` in case it is needed, be cautious to not override it. Also, for Android Drop-in `await AdyenDropIn.getReturnURL()` can be used to extract a `returnUrl`.
```

Reply:
Fixed [2777ef8](https://github.com/Adyen/adyen-react-native/pull/457/commits/2777ef86c1356e37ddaa7aa6437a375f7cfc2deb)